releng/org.eclipse.xtext.sdk.p2-repository/category.xml

```diff
@@ -6,5 +6,17 @@
 	<feature id="org.eclipse.xtend.sdk" version="2.14.0.qualifier">
 		<category name="Xtext"/>
 	</feature>
+	<bundle id="org.junit" version="0.0.0"></bundle>
```

Comment:
Do we need all these asm bundles?

Reply:
i dont know how we use asm. can you give details/ have a look?

Reply:
i'd be happy to remove as much as i can

Reply:
We do only use the core part of asm. Not sure what e.g. the XML stuff is about. Only asm and asm.tree are in our TP.

Reply:
for me it looks like pure asm is enhough. thats the only one thats installed. will update

Reply:
ok will readd tree